scripts/start.js

```diff
@@ -74,11 +77,12 @@ function setupCompiler (port) {
     clearConsole();
     const hasErrors = stats.hasErrors();
     const hasWarnings = stats.hasWarnings();
+
     if (!hasErrors && !hasWarnings) {
       console.log(chalk.green('Compiled successfully!'));
       console.log();
       // eslint-disable-next-line
-      console.log('The app is running at http://localhost:' + port + '/');
+      console.log('The app is running at http://localhost:' + port + basePath);
```

Comment:
well, needing to dig for the url for the embeded html was the most annoying part,
that's why i had to add this.. (regarding how was the project set up)